app/controllers/foreman_ansible/api/v2/job_invocations_controller_extensions.rb

```diff
@@ -0,0 +1,20 @@
+module ForemanAnsible
+  module Api
+    module V2
+      module JobInvocationsControllerExtensions
+        module ApipieExtensions
+          extend Apipie::DSL::Concern
+
+          opts = ::ForemanAnsible::AnsibleProvider.provider_inputs.find { |input| input.name == 'tags_flag' }.options.split("\n")
```

Comment:
Would this be worth adding to the REX PR?
We should probably go over all providers and add their respective parameters there, wouldn't it be easier?

Reply:
We are doing something along the lines for facets (note its not great example, because it doesn't work very well, but as a starting point )
https://github.com/theforeman/foreman/blob/fbf9bfef48fda7404e7b1b59fd0f1094bbd1a3ce/app/controllers/concerns/foreman/controller/parameters/host.rb#L28
https://github.com/theforeman/foreman/blob/fbf9bfef48fda7404e7b1b59fd0f1094bbd1a3ce/app/controllers/api/v2/hosts_controller.rb#L117